config.toml

```diff
@@ -0,0 +1,4 @@
+[[customizations.user]]
+name = "playtron"
+password = "playtron"
+groups = ["wheel"]
```

Comment:
I know we're not using bootc-image-builder yet but we should sync this with our Kickstart file before we forget. Just need to add the `input` and `tss` groups here.

Reply:
Adding those caused the build to fail because the groups did not yet exist. I'll take another look.